.github/ISSUE_TEMPLATE/new_bug.md

```diff
@@ -0,0 +1,32 @@
+---
+name: New Bug
+about: Identification of a new issue
+title: 'New Bug'
+labels: 
+assignees: '@abbylindsay'
+
+---
+
+**Describe the bug- why is it wrong?:**
+<!-- A clear and concise description of what you/the participant is seeing and why it is a problem. -->
+
+**How was this bug identified:**
+<!-- Who saw it? What was the context? -->
+
+**How to reproduce?:**
+<!-- If no instructions, you MUST provide an explaination as to why not. -->
+
+**Troubleshooting completed so far:**
+<!-- What has been done thus far to resolve/investigate the bug? -->
+
+**Expected behavior?:**
+<!-- Be descriptive: what should be happening? Provide examples if possible. -->
+
+**Screenshot(s):**
+<!-- Show the bug. -->
+
+**Priority level?:**
```

Comment:
This should be a drop down of predefined items.

```
- type: dropdown
    id: version
    attributes:
      label: Version
      description: What version of our software are you running?
      options:
        - 1.0.2 (Default)
        - 1.0.3 (Edge)
    validations:
      required: true
```

Reply:
[Reference](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms)

Reply:
we can have a drop down. rylee, marc, james and I had a thorough convo defining what type of issue is what level (1-5) that would be callibrated with the larger group

Reply:
yes, let's remove any ambiguity here and make this error proof and standardized.

Reply:
We should also define what each of the levels, etc. *mean* so it's clear.

Reply:
<!-- 1- a V&V will not be run for this issue alone. 2- Want to have 3- Causing minor impact to study/participant 4- Causing big impact to study/participant 5- BAD BAD BAD -->

this is listed under priority level to define. as i said, rylee, james, marc and I spent a lot of time thinking of a scale that we were going to share with the group so everyone is callibrated. the short version is listed here as a reminder.